src/clients/inst_autoconfigure.rb

```diff
@@ -40,8 +40,11 @@ def main
         Ops.get_map(Profile.current, ["general", "mode"], {})
       )
       @need_systemd_isolate = true
-      @max_steps = Ops.add(Builtins.size(Y2ModuleConfig.ModuleMap), 4)
+      final_restart_services = Ops.get_boolean(
+        Profile.current,["general", "mode", "final_restart_services"], true)
+      @max_steps = Ops.add(Builtins.size(Y2ModuleConfig.ModuleMap), 3)
```

Comment:
well, if you touch it, use ruby style: `@max_steps = Y2ModuleConfig.ModuleMap.size + 3` and maybe also please add comment why 3